File: 1323-maximum-69-number/1323-maximum-69-number.cpp
class Solution {
public:
    int maximum69Number (int num) {
        vector<int> d;
        while (num) {
            d.push_back(num % 10);
            num /= 10;
        }
        reverse(d.begin(), d.end());
        for (int i = 0; i < d.size(); ++i) {
            if (d[i] == 6) {
                d[i] = 9;
                break;
            }
        }
        int ans = 0;
        for (int i = 0; i < d.size(); ++i) {
            ans = (ans * 10) + d[i];
        }
        return ans;
    }
};

File: 1323-maximum-69-number/1323-maximum-69-number.py
class Solution:
    def maximum69Number (self, num: int) -> int:
        arr = []
        while (num > 0):
            arr.append(num % 10)
            num = num // 10
        arr.reverse()
        for x in range(len(arr)):
            if (arr[x] == 6):
                arr[x] = 9
                break
        ans = 0
        for x in range(len(arr)):
            ans = (ans * 10) + arr[x]
        return ans